src/dbnode/topology/consistency_level.go

```diff
@@ -373,3 +373,19 @@ func ReadConsistencyAchieved(
 	}
 	panic(fmt.Errorf("unrecognized consistency level: %s", level.String()))
 }
+
+// NumDesiredForReadConsistency returns the number of replicas that would ideally be used to
+// satisfy the read consistency.
+func NumDesiredForReadConsistency(level ReadConsistencyLevel, numReplicas, majority int) int {
+	switch level {
+	case ReadConsistencyLevelAll:
+		return numReplicas
+	case ReadConsistencyLevelMajority, ReadConsistencyLevelUnstrictMajority:
+		return majority
+	case ReadConsistencyLevelOne:
+		return 1
+	case ReadConsistencyLevelNone:
+		return 0
+	}
+	panic(fmt.Errorf("unrecognized consistency level: %s", level.String()))
```

Comment:
Why panic instead of returning an error here? Seems unidiomatic especially for just a simple helper function. If you really want to panic in the client session, it should catch the error thrown here and panic at the client session code.

Reply:
Alllright, seems like all other functions here panic instead of returning an error...